<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="UTF-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <meta name="robots" content="index, follow" />
    <meta name="description" content="Here a description" />
    <meta name="theme-color" content="#09f" />
    <link rel="stylesheet" href="css/normalize.css" />
    <link rel="stylesheet" href="css/style.css" />
    <link rel="canonical" href="https://diamondlab.com" lang="en" />
    <link
      rel="icon"
      href="assets/logos/diamond_lab_logo_favicon.svg"
      type="image/fav-icon"
    />
    <link rel="preconnect" href="https://fonts.googleapis.com" />
    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
    <script
      src="https://kit.fontawesome.com/744e5e3f2d.js"
      crossorigin="anonymous"
    ></script>
    <title>Diamond Lab</title>
  </head>
  <body>
    <div>
      <header>
        <div class="logo-content">
          <img src="assets/logos/diamond_lab_logo.svg" alt="diamond lab logo" />
          <h1 class="name-lab">Diamond Lab</h1>
        </div>
        <nav class="menu">
          <ul class="nav-links">
            <li class="icon_deco _1"><a href="index.html">Home</a></li>
            <li class="icon_deco _2 dropdown">
              <a href=""
                >Research<i class="fa-solid fa-caret-right"></i
                ><i class="fa-solid fa-caret-down"></i
              ></a>
              <ul class="dropdown_son">
                <li class="icon_deco _2.1">
                  <i class="fa-solid fa-bacterium fa-beat"></i
                  ><a href="community_editing.html">Community Editing</a>
                </li>
                <li class="icon_deco _2.2">
                  <i class="fa-solid fa-bacterium fa-beat"></i
                  ><a href="cyanobacterial_communities.html"
                    >Cyanobacterial Communities</a
                  >
                </li>
                <li class="icon_deco _2.3">
                  <i class="fa-solid fa-bacterium fa-beat"></i
                  ><a href="human_microbiome.html">Human Microbiome</a>
                </li>
              </ul>
            </li>
            <li class="icon_deco _3">
              <a href="publications.html">Publications</a>
            </li>
            <li class="icon_deco _4"><a href="news.html">News</a></li>
            <li class="icon_deco _5"><a href="team.html">Team</a></li>
            <li class="icon_deco _6">
              <a class="active" href="about_us.html">About</a>
            </li>
          </ul>
        </nav>
        <a class="btn-contact" href="contact.html">Contact</a>
      </header>

      <main class="container-main">
        <div class="about-us">
          <div class="about-top">
            <h1>Our Mission</h1>
            <p>
              Microbial communities in, on, and around us are required for the
              survival of all organisms on our planet and perform essential
              ecosystem services that maintain nutrient balances across the
              globe. The activities of microbes (bacteria, archaea, and fungi)
              within microbial communities exert critical controls on human,
              animal, and plant health, greenhouse gas flux, and have incredible
              potential for biotechnological applications. Importantly, no
              individual microorganism acts in isolation to perform these tasks.
              Instead, it is interaction networks between microbes within
              microbial communities that drive their real world impact through
              collective behavior.
            </p>
          </div>
          <div class="about-bottom">
            <p>
              The Diamond lab endeavors to understand how the features (organism
              interactions, composition, activity, metabolic pathways, genes,
              and mobile genetic elements) that make up microbial communities
              each individually contribute to their quantifiable collective
              behavior (i.e. community greenhouse gas production rate). To do so
              we rely heavily on the development and cultivation of model in
              vitro microbial communities that recapitulate the phenotypes
              observed in the natural systems they are derived from. Techniques
              then used by the Diamond lab to interogate in vitro community
              phenotypes include genome-resolved metagenomics, other
              high-throughput Oâ€™mics techniques, multivariate statical analysis,
              machine learning, and targeted genetic editing within microbial
              communities.
            </p>
            <p>
              As a member of the Innovative Genomics Institute BiomeForge
              initiative our ultimate goal is to understand the quantitative
              importance of microbial community features both to further
              understand microbial community biology as well as to enable the
              effective engineering of microbial communities in ways that will
              benifit human health, the environment, and society
            </p>
            <img
              class="img-about"
              src="assets/img_general/Macro_vs_Micro.jpg"
            />
          </div>
        </div>
      </main>

      <footer>
        <p class="copyright">
          <small>&copy; 2024 <b>Diamond Lab.</b> All Rights Reserved</small>
        </p>
        <div class="sponsors">
          <a href="https://www.berkeley.edu/research/" target="_blank"
            ><img
              class="logo-berkeley"
              src="assets/logos/University_of_California,_Berkeley-Logo.wine.png"
              alt="logo berkeley"
          /></a>
          <a href="https://innovativegenomics.org" target="_blank"
            ><img
              class="logo-igi"
              src="assets/logos/IGI-Logo-Institute.png"
              alt="logo innovative genomics institute"
          /></a>
        </div>
      </footer>
    </div>
  </body>
</html>
