<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="UTF-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <meta name="robots" content="index, follow" />
    <meta name="description" content="Here a description" />
    <meta name="theme-color" content="#09f" />
    <link rel="stylesheet" href="css/normalize.css" />
    <link rel="stylesheet" href="css/style.css" />
    <link rel="canonical" href="https://diamondlab.com" lang="en" />
    <link
      rel="icon"
      href="assets/logos/diamond_lab_logo_favicon.svg"
      type="image/fav-icon"
    />
    <link rel="preconnect" href="https://fonts.googleapis.com" />
    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
    <script
      src="https://kit.fontawesome.com/744e5e3f2d.js"
      crossorigin="anonymous"
    ></script>
    <title>Diamond Lab</title>
  </head>
  <body>
    <div class="container_body">
      <header>
        <div class="logo-content">
          <img src="assets/logos/diamond_lab_logo.svg" alt="diamond lab logo" />
          <h1 class="name-lab">Diamond Lab</h1>
        </div>
        <nav class="menu">
          <ul class="nav-links">
            <li class="icon_deco _1"><a href="index.html">Home</a></li>
            <li class="icon_deco _2 dropdown">
              <a class="active" href=""
                >Research<i class="fa-solid fa-caret-right"></i
                ><i class="fa-solid fa-caret-down"></i
              ></a>
              <ul class="dropdown_son">
                <li class="icon_deco _2.1">
                  <i class="fa-solid fa-bacterium fa-beat"></i
                  ><a href="community_editing.html">Community Editing</a>
                </li>
                <li class="icon_deco _2.2">
                  <i class="fa-solid fa-bacterium fa-beat"></i
                  ><a href="cyanobacterial_communities.html"
                    >Cyanobacterial Communities</a
                  >
                </li>
                <li class="icon_deco _2.3">
                  <i class="fa-solid fa-bacterium fa-beat"></i
                  ><a href="human_microbiome.html">Human Microbiome</a>
                </li>
              </ul>
            </li>
            <li class="icon_deco _3">
              <a href="publications.html">Publications</a>
            </li>
            <li class="icon_deco _4"><a href="news.html">News</a></li>
            <li class="icon_deco _5"><a href="team.html">Team</a></li>
            <li class="icon_deco _6"><a href="about_us.html">About</a></li>
          </ul>
        </nav>
        <a class="btn-contact" href="contact.html">Contact</a>
      </header>
      <main class="container-main">
        <article class="container-research">
          <h1>Community Editing</h1>
          <section class="overview">
            <h2>Overview</h2>
            <p>
              Microbes, microscopic single celled organisms, are essential for
              the survival of humans, animals, plants, and Earth’s ecosystems.
              <strong
                >However, no individual microorganism acts in isolation to
                perform these tasks. Instead, microbes interact in communities,
                and their real-world impact is realized only through their
                collective behavior</strong
              >. Thus, to truly understand how microbes work we must develop the
              specialized tools needed to study them within community contexts.
            </p>
            <p>
              The ability to edit the DNA of isolated microorganisms has been
              foundational to increasing our understanding of how they work. As
              an example, if a microbe eats sucrose and you want to know what
              genes are responsible for this behavior, one could edit the DNA of
              that microbe to disrupt different genes until you break one and
              the microbe can no longer eat sucrose. In this way scientists have
              used DNA editing to learn about how genes function and control
              microbial behavior. However, until now, this type of DNA editing
              approach has required isolated microorganisms that can grow in the
              lab. This is problematic from a community context for three
              critical reasons:
            </p>
            <ol>
              <li>
                Most microbes (~99%) have not been isolated in laboratory
                settings, the isolation of microbes from many novel bacterial
                phyla has proven extremely difficult, and it is known that
                growth in community contexts can enable the cultivation of at
                least some of these microbes in lab-based communities.
              </li>
              <li>
                The behavior of an individual microorganism in a community
                context is different than in isolation due to interactions
                between the constituent species.
              </li>
              <li>
                The collective behavior of microbial communities is a result of
                a combination of many interactions, and studying these
                combinations out of a community context is extremely difficult.
              </li>
            </ol>
            <img
              class="cm_fig2"
              src="assets/research_img/com_edit/Com_Edit_Page_Fig1.jpg"
            />
            <p>
              Our lab is involved in a large Innovative Genomics Institute
              collaborative effort to develop tools for in situ targeted genetic
              modification and interrogation of microbial communities known as
              <strong>BiomeForge</strong>. Our primary research foci within this
              collaboration are the development of in vitro model microbiome
              systems, modeling and characterization of in vitro model
              microbiomes, and development of informatic toolsets for targeted
              community editing approaches.
            </p>
          </section>
          <section class="aproach">
            <h2>Approach</h2>
            <p>
              The technologies necessary to bring DNA editing to microbial
              communities has been pioneered at the IGI. The CRISPR-Cas
              revolution originating in Professor Jennifer Doudna’s lab provides
              the tools necessary to bring traditional targeted genetic editing
              to communities. The development of genome-resolved metagenomics
              (an approach to reconstruct all the individual microbial genomes
              in a complex community) by Jill Banfield’s lab, provides the
              understanding required to guide editing and characterize its
              effects. We have recently merged these complementary skill sets in
              demonstrating, for the first time, generalizable methods to
              directly edit DNA in specific microorganisms while in community
              contexts. This novel approach can be explained at its core as a
              three-step process:
            </p>
            <ol>
              <li>
                <strong>Genomic Characterization:</strong> First a microbial
                community is sequenced and the genomes of the organisms present
                are assembled to understand the community composition and
                provide the template for editing.
              </li>
              <li>
                <strong>Assess Delivery Methods:</strong> We use a technology
                known as Environmental Transformation Sequencing
                (<strong>ET-Seq</strong>; described below) to assess how well a
                specific DNA delivery approach can get DNA into microbes of this
                community.
              </li>
              <li>
                <strong>Make Targeted Edits:</strong> We use a DNA-editing
                All-in-one RNA-guided CRISPR-Cas Transposase
                (<strong>DART</strong>; described below) to then make targeted
                DNA insertions into a microbe of interest using the best DNA
                delivery method identified by <strong>ET-Seq</strong>.
              </li>
            </ol>
            <img
              class="cm_fig2"
              src="assets/research_img/com_edit/Com_Edit_Page_Fig2.jpg"
            />
            <p>
              While it is still early days for the development and application
              of community editing, the technique has nonetheless demonstrated
              some very exciting results. Using this technology, it was possible
              to engineer a metabolic function into a community (lactose
              utilization), without doing any isolation-based engineering, that
              was previously absent from all the members. This allowed the
              targeted member to be enriched significantly when lactose was
              present in the medium (lower left panel). Also, we have found that
              the targeting of DART is extremely specific, allowing for
              strain-specific selection and isolation of two different E. coli
              strains out of a complex infant microbiome enrichment culture,
              where no member had been previously isolated (lower right panel).
            </p>
            <img
              class="cm_fig3"
              src="assets/research_img/com_edit/Com_Edit_Page_Fig3.jpg"
            />
          </section>
          <section class="technologies">
            <h2>Specific technologies</h2>
            <p>
              <strong>ET-Seq</strong>: Editing organisms in a complex microbiome
              requires knowledge of which members are accessible to nucleic acid
              delivery and editing. Environmental Transformation Sequencing
              (ET-Seq) is a technology developed to assess the ability of
              individual species within a microbial community to acquire and
              integrate exogenous DNA without the need to isolate any individual
              members. In ET-seq, a microbial community is exposed to a randomly
              integrating mobile genetic element (an un-targeted transposon)
              and, in the absence of any selection, total community DNA is
              extracted and sequenced using two protocols. In the first, we
              enrich and sequence the junctions between the inserted and host
              DNA to determine insertion location and quantity in each host. In
              the second protocol, we use low-depth metagenomic sequencing to
              quantify the abundance of each community member in a sample.
              Together, the data from these two sequencing procedures are
              analyzed with an informatic toolset we developed for this purpose,
              ETSuite, and a relative measure of transformation efficiency for
              each of the community members is output.
            </p>
            <img
              class="cm_fig4"
              src="assets/research_img/com_edit/Com_Edit_Page_Fig4.jpg"
            />
            <p>
              <strong>DART</strong>: Once it is determined that an organism in a
              microbial community can acquire and integrate exogenous DNA, a
              method is necessary to target DNA delivered to the community to
              that specific organism and only that organism. For this we have
              developed DNA-editing all-in-one RNA-guided CRISPR–Cas transposase
              (DART) systems for locus-specific insertion of DNA into organisms
              within a microbial community. The DART system uses an RNA-guided
              CRISPR–Cas Tn7 transposase to deliver cargo DNA that is between
              transposon repeats on a plasmid into a specific location of a
              target genome defined by the guide RNA (gRNA) also encoded on the
              same plasmid. These systems also include barcodes that make them
              compatible with ET-Seq to allow for the detection and tracking of
              uniquely edited cells. Thus, DART can be used seamlessly with
              ET-seq for rapid assay of the efficacy of CRISPR–Cas-guided
              transposition into the genome of a target organism in the absence
              of selectable markers.
            </p>
            <img
              class="cm_fig5"
              src="assets/research_img/com_edit/Com_Edit_Page_Fig5.png"
            />
          </section>
          <section>
            <h3>Selected publications</h3>
            <div class="research-pubs">
              <ul>
                <li class="publication-item-research">
                  <span>
                    Rubin, B. E., Diamond, S., Cress, B. F., Crits-Christoph,
                    A., Lou, Y. C., Borges, A. L., Shivram, H., He, C., Xu, M.,
                    Zhou, Z., Smith, S. J., Rovinsky, R., Smock, D. C. J., Tang,
                    K., Owens, T. K., Krishnappa, N., Sachdeva, R., Barrangou,
                    R., Deutschbauer, A. M., Banfield, J. F., … Doudna, J. A.
                    (2022).<br /><span class="pub-title"
                      >Species- and site-specific genome editing in complex
                      bacterial communities. Nature microbiology, 7(1),
                      34–47.</span
                    ><a
                      href="https://doi.org/10.1038/s41564-021-01014-7"
                      target="_blank"
                      >(https://doi.org/10.1038/s41564-021-01014-7)</a
                    >
                  </span>
                </li>
              </ul>
            </div>
          </section>
          <section class="funding">
            <h3>Funding</h3>
            <div class="funding-logos com-ed">
              <a href="https://innovativegenomics.org"
                ><img
                  class="logo-igi"
                  src="assets/logos/IGI-Logo-Institute.png"
                  alt="logo innovative genomics institute"
              /></a>
              <a href="https://mcafes.lbl.gov"
                ><img
                  class="logo-m-cafe"
                  src="assets/logos/m-CAFEs.png"
                  alt="m-CAFEs logo"
              /></a>
              <a href="https://www.lbl.gov"
                ><img
                  class="logo-lbl"
                  src="assets/logos/Berkley_Lab_logo.png"
                  alt="Berkley Lab logo"
              /></a>
            </div>
          </section>
        </article>
      </main>
      <footer>
        <p class="copyright">
          <small>&copy; 2024 <b>Diamond Lab.</b> All Rights Reserved</small>
        </p>
        <div class="sponsors">
          <a href="https://www.berkeley.edu/research/" target="_blank"
            ><img
              class="logo-berkeley"
              src="assets/logos/University_of_California,_Berkeley-Logo.wine.png"
              alt="logo berkeley"
          /></a>
          <a href="https://innovativegenomics.org" target="_blank"
            ><img
              class="logo-igi"
              src="assets/logos/IGI-Logo-Institute.png"
              alt="logo innovative genomics institute"
          /></a>
        </div>
      </footer>
    </div>
  </body>
</html>
