<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta name="robots" content="index, follow">
    <meta name="description" content="Here a description">
    <meta name="theme-color" content="#09f">
    <link rel="stylesheet" href="css/normalize.css">
    <link rel="stylesheet" href="css/style.css">
    <link rel="canonical" href="https://diamondlab.com" lang="en">
    <link rel="icon" href="assets/logos/diamond_lab_logo_favicon.svg" type="image/fav-icon">
    <link rel="preconnect" href="https://fonts.googleapis.com">
    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
    <script src="https://kit.fontawesome.com/744e5e3f2d.js" crossorigin="anonymous"></script>
    <title>Diamond Lab</title>
</head>
<body>
    <div  class="container_body">

        <header>
            <div class="logo-content">
                <img src="assets/logos/diamond_lab_logo.svg" alt="diamond lab logo">
                <h1 class="name-lab">Diamond Lab</h1>
            </div>
            <nav class="menu">
                <ul class="nav-links">
                    <li class="icon_deco _1"><a href="index.html">Home</a></li>
                    <li class="icon_deco _2 dropdown"><a class="active" href="">Research<i class="fa-solid fa-caret-right"></i><i class="fa-solid fa-caret-down"></i></a>
                        <ul class="dropdown_son">
                            <li class="icon_deco _2.1"><i class="fa-solid fa-bacterium fa-beat"></i><a href="community_editing.html">Community Editing</a></li>
                            <li class="icon_deco _2.2"><i class="fa-solid fa-bacterium fa-beat"></i><a href="cyanobacterial_communities.html">Cyanobacterial Communities</a></li>
                            <li class="icon_deco _2.3"><i class="fa-solid fa-bacterium fa-beat"></i><a href="human_microbiome.html">Human Microbiome</a></li>
                        </ul>
                    </li>
                    <li class="icon_deco _3"><a href="publications.html">Publications</a></li>
                    <li class="icon_deco _4"><a href="news.html">News</a></li>
                    <li class="icon_deco _5"><a href="team.html">Team</a></li>
                    <li class="icon_deco _6"><a href="about_us.html">About</a></li>
                </ul>
            </nav>
            <a class="btn-contact" href="contact.html">Contact</a>
        </header>
        <main  class="container-main">
            <article class="container-research">
                <h1>Cyanobacterial Communities</h1>
                <section class="overview">    
                    <h2>Overview</h2>
                    <p>Cyanobacteria, also known as blue-green algae, are an extremely important, abundant, and widespread group of photosynthetic microorganisms found in almost every environment on our planet. They are critically important for Earth’s carbon and nitrogen cycles as they remove as much carbon dioxide (CO2) from the air as all the world’s plants, produce ~30% of our breathable oxygen, provide up to 50% of the fixed nitrogen other organisms use, and both directly produce and influence methane release from aquatic environments. Cyanobacteria have also been intensively studied for their industrial and biotechnological potential, as engineered cyanobacteria have shown the potential to produce a variety of fuels, chemicals, and medicines using only light, CO2, and micronutrients as an input. However, the impacts of cyanobacteria are not always friendly, as they are well known to produce a variety of natural products that include potent neurotoxins. Cyanobacterial blooms producing these toxins can impair the safety of limited fresh water supplies used for drinking, irrigation, fishing, and recreation. Importantly, it is predicted that climate change and anthropogenetic eutrophication will have major impacts on cyanobacteria in natural systems. <strong>Thus, it is critical that we study these incredibly impactful organisms in the context of their microbial communities to understand how they perform ecosystem functions, how they can be leveraged for biotechnology, and how they will be impacted by human driven changes globally.</strong></p>
                    <img class="cy_ fig1" src="assets/research_img/cyanob_comm/Cyano_Coms_Page_Fig1.png">
                    <p>Being primary producers, Cyanobacteria form the bedrock of microbial food chains in many environments, and readily form complex interactive assemblages with other environmental microorganisms. This is similar to how microbes assemble around plant roots in an area called the rhizosphere, whereas the analogous area surrounding a cyanobacterium is known as the phycosphere. Cyanobacterial communities range in structure from planktonic assemblages to highly organized biofilms in streams and desert crusts. We now know that the co-occurring organisms in these communities interact with cyanobacterial cells and exert significant impacts on their growth and physiology. However, the specific co-occurring organisms, genes, and interactions that drive the ultimate collective behavior of these communities remain little understood. <strong>More broadly, the study of these and other microbial communities is hampered by the fact that few, if any, model systems exist that enable the systematic study of the mechanisms driving the collective behavior of these communities</strong></p>. Thus we are interested in the development of model cyanobacterial community systems and targeted community editing tools to interrogate the functions of these systems in the context of environmental sustainability and human health.</p>
                    <img class="cy_ fig2" src="assets/research_img/cyanob_comm/Cyano_Coms_Page_Fig2.png">
                </section>
                <section class="aproach">
                    <h2>Approach</h2>
                    <p>Cyanobacterial communities are inherently excellent systems to explore and develop in vitro microbial community models. By their very nature, cyanobacteria naturally form microbial assemblages, and it is widely acknowledged in the field that obtaining axenic isolates of cyanobacteria is difficult since they strongly associate with, and are hard to separate from, microorganisms present in the source material from where they are collected. Unfortunately, while this is widely known, little to no work has been done to systematically characterize the communities present in non-axenic isolates. We hope to leverage this property, as well as other biotechnologically favorable traits of cyanobacteria including ease of growth, natural transformability, and unique metabolic traits to construct, derive, and engineer cyanobacterial communities in ways that enable us to study how the organisms, genes, and interactions in these communities drive their responses to environmental conditions and ultimately their collective behavior. Currently we use two approaches to create cyanobacterial communities</p>
                    <ol>
                        <li>We create semi-synthetic consortia where a single axenic cyanobacterium is inoculated with environmentally derived bacteria and passaged until a stable community forms (<b>UniComs</b>)</li>
                        <li>We use single filament isolation to collect a single cyanobacterial filament with its physically associated microbes, and grow and passage the filament until a stable community forms (<b>MatComs</b>)</li>
                    </ol>
                    <img class="cy_ fig3" src="assets/research_img/cyanob_comm/Cyano_Coms_Page_Fig3.png">
                    <p>Since systematic construction of model in vitro microbial communities is still a very unexplored space, one major goal of our lab is to understand the factors that impact stable community formation as well as the compositions of the communities that can be produced using these methods. Subsequently we are interested in assessing the impact of nutrient concentrations, stress, and the behavior of cyanobacteria engineered to produce biotechnologically useful products in the context of whole communities. The methods we use for interrogation of these communities include: microscopy, biochemical and physiological measurements, genome resolved metagenomic sequencing, and transcriptomics. We are also focusing on developing targeted genetic tools to enable direct genetic modification of these communities (Community Editing) by identifying and harvesting natural genetic elements within these systems that may enable DNA mobilization and delivery.</p>
                    <img class="cy_ fig4" src="assets/research_img/cyanob_comm/Cyano_Coms_Page_Fig4.png">
                </section>
                <section>
                    <h3>Selected publications</h3>
                    <ul>
                        <li><span>X<br /><a href="#" target="_blank"></a></span></li>
                    </ul>
                </section>
                <section class="funding">
                    <h3>Funding</h3>
                    <div class="funding-logos cyano">
                    <a href="https://innovativegenomics.org"><img class="logo-igi" src="assets/logos/IGI-Logo-Institute.png" alt="logo innovative genomics institute"></a>
                    <a href="https://www.moore.org"><img class="logo_gmoore" src="assets/logos/Gordon_and_Betty_Moore_logo.png" alt="gordon and betty logo"></a>
                    </div>
                </section>
            </article>
        </main>
        <footer>
            <p class="copyright"><small>&copy; 2024 <b>Diamond Lab.</b> All Rights Reserved</small></p>             
            <div class="sponsors">
                <a href="https://www.berkeley.edu/research/" target="_blank"><img class="logo-berkeley" src="assets/logos/University_of_California,_Berkeley-Logo.wine.png" alt="logo berkeley"></a>
                <a href="https://innovativegenomics.org" target="_blank"><img class="logo-igi" src="assets/logos/IGI-Logo-Institute.png" alt="logo innovative genomics institute"></a>
            </div>     
        </footer>

    </div>   
</body>
</html>
