<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="UTF-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <meta name="robots" content="index, follow" />
    <meta name="description" content="Here a description" />
    <meta name="theme-color" content="#09f" />
    <link rel="stylesheet" href="css/normalize.css" />
    <link rel="stylesheet" href="css/style.css" />
    <link rel="canonical" href="https://diamondlab.com" lang="en" />
    <link
      rel="icon"
      href="assets/logos/diamond_lab_logo_favicon.svg"
      type="image/fav-icon"
    />
    <link rel="preconnect" href="https://fonts.googleapis.com" />
    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
    <script
      src="https://kit.fontawesome.com/744e5e3f2d.js"
      crossorigin="anonymous"
    ></script>
    <title>Diamond Lab</title>
  </head>
  <body>
    <div class="container_body">
      <header>
        <div class="logo-content">
          <img src="assets/logos/diamond_lab_logo.svg" alt="diamond lab logo" />
          <h1 class="name-lab">Diamond Lab</h1>
        </div>
        <nav class="menu">
          <ul class="nav-links">
            <li class="icon_deco _1"><a href="index.html">Home</a></li>
            <li class="icon_deco _2 dropdown">
              <a class="active" href=""
                >Research<i class="fa-solid fa-caret-right"></i
                ><i class="fa-solid fa-caret-down"></i
              ></a>
              <ul class="dropdown_son">
                <li class="icon_deco _2.1">
                  <i class="fa-solid fa-bacterium fa-beat"></i
                  ><a href="community_editing.html">Community Editing</a>
                </li>
                <li class="icon_deco _2.2">
                  <i class="fa-solid fa-bacterium fa-beat"></i
                  ><a href="cyanobacterial_communities.html"
                    >Cyanobacterial Communities</a
                  >
                </li>
                <li class="icon_deco _2.3">
                  <i class="fa-solid fa-bacterium fa-beat"></i
                  ><a href="human_microbiome.html">Human Microbiome</a>
                </li>
              </ul>
            </li>
            <li class="icon_deco _3">
              <a href="publications.html">Publications</a>
            </li>
            <li class="icon_deco _4"><a href="news.html">News</a></li>
            <li class="icon_deco _5"><a href="team.html">Team</a></li>
            <li class="icon_deco _6"><a href="about_us.html">About</a></li>
          </ul>
        </nav>
        <a class="btn-contact" href="contact.html">Contact</a>
      </header>
      <main class="container-main">
        <article class="container-research">
          <h1>Human Microbiome</h1>
          <section class="overview">
            <h2>Overview</h2>
            <p>
              Human health is directly connected to the roughly 100 trillion
              microbes that live in our digestive tract (The Human Microbiome).
              A flurry of research on the human microbiome has revealed that
              microbes in our gut may directly impact a diverse range of human
              health issues from psychological conditions to cardiovascular
              disease. However, the microbial diversity present across even
              healthy adult human microbiomes is immense, still poorly
              characterized in some sub-populations, and is directly impacted by
              host-unique features such as age, diet, antibiotic usage, and
              underlying medical conditions. This high level of individualized
              diversity has hindered the establishment of cause-and-effect
              relationships between microbiome compositions and human health.
              Thus, several fundamental challenges and knowledge gaps remain in
              our ability to rapidly assess diverse populations and identify
              features (e.g. genes, pathways, microbes, and interactions) at
              sufficiently high levels of resolution to drive mechanistic
              insights including that:
            </p>
            <ol>
              <li>
                Experiments on human or animal subjects are complex and
                expensive leading to smaller sample sizes and underpowered
                statistics.
              </li>
              <li>
                Studies based on direct stool sampling are an indirect measure
                and often not reflective of true internal microbiome responses.
              </li>
              <li>
                Typical 16s rRNA amplicon-based stool sequencing studies do not
                capture features such as microbial metabolism, strains, or
                mobile elements (e.g. plasmids and phages), which may be the
                features driving health outcomes.
              </li>
            </ol>
            <p>
              We hope to overcome these issues through the development and
              genome-resolved metagenomic characterization of in vitro human
              microbiome systems that are reflective of host microbiomes and
              recapitulate host phenotypic responses. As development of these
              systems is in the early stages, it is likely that additional
              modifications to growth conditions will result in in vitro
              communities that even more realistically capture phenotypes of
              their hosts. By using in vitro community cultivation we can
              explore an extremely wide set of growth conditions and
              experimental perturbations without the need for human or animal
              subjects.
            </p>
            <img
              class="hm_fig"
              src="assets/research_img/human_micro/HMB_Page_Fig1.png"
            />
          </section>
          <section class="research-area">
            <h2>Research Areas</h2>
            <p>
              Microbial Persistence: Persistence, the ability of a microorganism
              to enter or stably remain within a microbial community, is a basic
              and important property of microbiome dynamics. Understanding the
              underlying features (i.e., genes and metabolic pathways)
              controlling microbial persistence within microbiomes is relevant
              for both basic biological understanding and practical
              applications. This includes understanding the dynamics of
              microbial succession in communities, pathogenic invasion into
              communities, persistence of probiotic strains in human
              microbiomes, and enabling engineered organisms to persist in
              natural microbiomes.
            </p>
            <p>
              In this project we are challenging stable in vitro human
              microbiomes with externally introduced probiotic strains (e.g.
              Lactobacillus sp.) as models to uncover the features (e.g. genes
              and metabolic pathways) impacting the persistence of an introduced
              strain into a stable system. Using the in vitro strategy will also
              allow a significant number of stable community growth conditions
              and probiotic strains to be tested on each unique human microbiome
              sample.
            </p>
            <img
              class="hm_fig"
              src="assets/research_img/human_micro/HMB_Page_Fig2.png"
            />
          </section>
          <section class="aproach">
            <h2>Approach</h2>
            <p>
              While our lab has several research areas of interest within the
              human (and other) microbiome space, our general approach to
              cultivation and feature identification across these systems is
              roughly congruent and follows a four-step strategy:
            </p>
            <ol>
              <li>
                Cultivate many biological and technical replicate microbiomes in
                parallel using high-throughput techniques and passage until a
                stable community composition is achieved.
              </li>
              <li>
                Apply an experimental perturbation to stable communities,
                sequence source and post-perturbation communities using shotgun
                metagenomic sequencing, and assemble microbial genomes and other
                genetic elements using genome-resolved metagenomic methods.
              </li>
              <li>
                Apply classical statistical analysis and machine learning
                methods to identify features (e.g. genes, metabolic pathways,
                organism compositions, ect.) within communities that are
                predictive of measured outcomes (e.g. persistence duration).
              </li>
              <li>
                Work with collaborators within BiomeForge to validate feature
                impacts on tested perturbation using targeted community editing
                methods.
              </li>
            </ol>
            <img
              class="hm_fig"
              src="assets/research_img/human_micro/HMB_Page_Fig3.png"
            />
          </section>
          <section>
            <h3>Selected publications</h3>
            <div>
              <ul>
                <li class="publication-item">
                  <span>
                    Lou, Y. C., Olm, M. R., Diamond, S., Crits-Christoph, A.,
                    Firek, B. A., Baker, R., Morowitz, M. J., & Banfield, J. F.
                    (2021). Infant gut strain persistence is associated with
                    maternal origin, phylogeny, and traits including surface
                    adhesion and iron acquisition.<br /><a
                      href="https://doi.org/10.1016/j.xcrm.2021.100393"
                      target="_blank"
                      >(https://doi.org/10.1016/j.xcrm.2021.100393)</a
                    >
                  </span>
                </li>
              </ul>
            </div>
          </section>
          <section class="funding">
            <h3>Funding</h3>
            <div class="funding-logos hum-micr">
              <a href="https://innovativegenomics.org"
                ><img
                  class="logo-igi"
                  src="assets/logos/IGI-Logo-Institute.png"
                  alt="logo innovative genomics institute"
              /></a>
              <a href="https://curcifoundation.org"
                ><img
                  class="logo-shurl"
                  src="assets/logos/shurl-and-kay-curci-logo.png"
                  alt="shurl andkay curci logo"
              /></a>
            </div>
          </section>
        </article>
      </main>
      <footer>
        <p class="copyright">
          <small>&copy; 2024 <b>Diamond Lab.</b> All Rights Reserved</small>
        </p>
        <div class="sponsors">
          <a href="https://www.berkeley.edu/research/" target="_blank"
            ><img
              class="logo-berkeley"
              src="assets/logos/University_of_California,_Berkeley-Logo.wine.png"
              alt="logo berkeley"
          /></a>
          <a href="https://innovativegenomics.org" target="_blank"
            ><img
              class="logo-igi"
              src="assets/logos/IGI-Logo-Institute.png"
              alt="logo innovative genomics institute"
          /></a>
        </div>
      </footer>
    </div>
  </body>
</html>
